src/scss/components/Card-desktop.scss

```
@@ -0,0 +1,25 @@
+@use "../abstracts/fonts.scss";
+@use "../abstracts/colors.scss";
+@import "../abstracts/breakpoints.scss";
+
+.product-card {
+  @include desktop {
+    flex-direction: row;
+    height: 12rem;
+    justify-content: flex-start;
+    width: 34rem;
+  }
+
+  .product-card__picture {
```

Zgonie z BEM w skompilowanym CSS-ie powinieneś mieć selektor `.product-card__picture`, nie `.product-card .product-card__picture`:
```suggestion
  &__picture {
```